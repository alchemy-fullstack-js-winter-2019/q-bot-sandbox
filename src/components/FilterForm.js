import React from 'react';

export default function FilterForm({ value, onChange }) {
  return (
    <input type="text" name="searchTerm" value={value} onChange={onChange} />
  )
}
