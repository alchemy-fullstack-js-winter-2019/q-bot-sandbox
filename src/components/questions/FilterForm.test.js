import React from 'react';
import { shallow } from 'enzyme';
import FilterForm from './FilterForm';

describe('FilterForm', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(
      <FilterForm value={'TA'} onChange={jest.fn()}/>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
