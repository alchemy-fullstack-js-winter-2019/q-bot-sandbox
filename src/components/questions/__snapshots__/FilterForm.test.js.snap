// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterForm matches a snapshot 1`] = `
<Fragment>
  <h2
    className="addFilterHeader"
  >
    Filter & Sort 
  </h2>
  <label
    className="questionLabel"
  >
    Question:
  </label>
  <input
    name="searchTerm"
    onChange={[MockFunction]}
    type="text"
    value="TA"
  />
</Fragment>
`;
